src/sentry/receivers/outbox/control.py

```diff
@@ -68,22 +69,30 @@ def process_async_webhooks(payload: Mapping[str, Any], region_name: str, **kwds:
 
     if SiloMode.get_current_mode() == SiloMode.CONTROL:
         # By default, these clients will raise errors on non-20x response codes
-        response = RegionSiloClient(region=region).request(
-            method=webhook_payload.method,
-            path=webhook_payload.path,
-            headers=webhook_payload.headers,
-            # We need to send the body as raw bytes to avoid interfering with webhook signatures
-            data=webhook_payload.body,
-            json=False,
-        )
-        logger.info(
-            "webhook_proxy.complete",
-            extra={
-                "status": response.status_code,
-                "request_path": webhook_payload.path,
-                "request_method": webhook_payload.method,
-            },
-        )
+        try:
+            response = RegionSiloClient(region=region).request(
+                method=webhook_payload.method,
+                path=webhook_payload.path,
+                headers=webhook_payload.headers,
+                # We need to send the body as raw bytes to avoid interfering with webhook signatures
+                data=webhook_payload.body,
+                json=False,
+            )
+            logger.info(
+                "webhook_proxy.complete",
+                extra={
+                    "status": getattr(
+                        response, "status_code", 204
```

Comment:
This was super quirky, but realized this was causing integration outboxes to be backlogged incorrectly. `status_code` isn't present on the return type for any 204 response, as it returns `{}` instead of a `Response` type.